src/madhuraMJun20/assignment30Plus/Coding_Exam_10.java

```diff
@@ -0,0 +1,35 @@
+/*Print minimum rotation required to compare both strings equality.
+Example 1  : 
+String input1 = "abcdef";
+String input2 = "defabc";
+output : 3
+
+Example 2  : 
+String input1 = "adbcdef"; -> push right
+fadbcde
+efadbcd
+defadbc
+String input2 = "defadbc";
+output : 3*/
+package madhuraMJun20.assignment30Plus;
+
+public class Coding_Exam_10 {
+	static int getMinimumRotation(String input1, String input2) {
+		int rotationCount = 0;
+		if (input1.length() == input2.length()) {
+			while (!input1.equals(input2)) {
```

Comment:
Just a suggestion. what if String can't be made equals even after all possible rotation ?? this will run infinitely ...

you can handled it by adding one more condition.
while(!input1.equals(input2) && rotationCount< input1.length)